code/game/striketeams/nanotrasen_deathsquad.dm

```diff
@@ -121,10 +122,14 @@
 
 	var/obj/item/weapon/card/id/W = new(src)
 	W.name = "[real_name]'s ID Card"
-	W.icon_state = "centcom"
-	W.access = get_centcom_access("Death Commando")
-	W.icon_state = "deathsquad"
-	W.assignment = "Death Commando"
+	if(leader)
+		W.access = get_centcom_access("Creed Commander")
+		W.icon_state = "creed"
+		W.assignment = "Creed Commander"
```

Comment:
Is this supposed to be "Commander Creed"?

Reply:
nope

Reply:
It's only half of a WH40k reference

Reply:
tell me what to make it and i will